src/ds/d400/d400-device.cpp

```diff
@@ -1081,7 +1101,28 @@ namespace librealsense
         //TODO: Implement
     }
 
-    platform::usb_spec d400_device::get_usb_spec() const
+    // Check if need change camera name due to number modifications on one device PID.
+    void update_device_name(std::string& device_name, const ds::d400_caps cap, const uint8_t cur_gvd_version)
+    {
+		switch (cap)
+		{
+		    case ds::d400_caps::CAP_IR_FILTER:
+			    if (is_capability_supports(cap, cur_gvd_version))
```

Comment:
No need to check it inside every case,
Lets take it out and only call `update_device_name` if it's true

Reply:
The code is optimized.